src/redux/books/categories/CategoriesSlice.jsx

```diff
@@ -0,0 +1,19 @@
+/** ****************************************
```

Comment:
Please follow the ducks pattern as requested, move the categories folder out of books folder. It is one of the best practices for folder system used in React projects 👍 
<img width="1085" alt="Screenshot 2023-04-12 at 9 10 53 PM" src="https://user-images.githubusercontent.com/25887253/231547335-b135ed0e-a831-4e03-bb5c-18b422ff710d.png">